### ISSUE TITLE
Logits dimensions

### ISSUE THREAD

**Issue by matus-pikuliak** [guillaumegenthial/sequence_tagging#41]
Hey,
I was looking at your model and I noticed that you have the logits coming out of your LSTM set to the number of classes (W.shape = [2*lstm, ntags]). Is there any reason to do so when you are using CRF to induce the tags?

**Comment by cs230-stanford** [comment]
@matus-pikuliak 
I was under the understanding that the CRF API required a tensor of shape `ntags` but I might be wrong !
see here : https://www.tensorflow.org/api_docs/python/tf/contrib/crf/crf_sequence_score

**Comment by guillaumegenthial** [comment]
closing due to inactivity.
